config/bootstrap.js

```diff
@@ -1,16 +1,14 @@
 /**
- * Bootstrap
+ * Seed Function
  * (sails.config.bootstrap)
  *
- * An asynchronous bootstrap function that runs before your Sails app gets lifted.
- * This gives you an opportunity to set up your data model, run jobs, or perform some special logic.
+ * A function that runs just before your Sails app gets lifted.
+ * > Need more flexibility?  You can also create a hook.
  *
- * For more information on bootstrapping your app, check out:
- * http://sailsjs.org/#!/documentation/reference/sails.config/sails.config.bootstrap.html
+ * For more information on seeding your app with fake data, check out:
+ * https://sailsjs.com/config/bootstrap
  */
 
-module.exports.bootstrap = function (cb) {
-  // It's very important to trigger this callback method when you are finished
-  // with the bootstrap!  (otherwise your server will never lift, since it's waiting on the bootstrap)
-  cb();
+module.exports.bootstrap = async function () {
+  //
```

Comment:
As far as I can tell you no longer need the callback function.

Reply:
I'll poke around